playwright.config.js

```diff
@@ -10,13 +11,17 @@ require("dotenv").config();
 /**
  * @see https://playwright.dev/docs/test-configuration
  */
+export const STORAGE_STATE = path.join(__dirname, "states/.auth/user.json");
+
+console.log("Storage state path:", STORAGE_STATE);
+
 module.exports = defineConfig({
```

Comment:
```suggestion
export default defineConfig({
```



ось в цьому проблема, тут намішано commonJS + modules. Через це експорт export const STORAGE_STATE просто напросто не спрацьовує і в файлі login-setup.js немає доступу до змінної STORAGE_STATE